src/services/goals.js

```diff
@@ -106,35 +107,33 @@ export async function saveGoalsForReport(goals, report, transaction) {
   await removeUnusedObjectivesGoalsFromReport(report.id, goals, transaction);
 
   return Promise.all(goals.map(async (goal) => {
-    let goalId = goal.id;
+    const goalId = goal.id;
+    const fields = goal;
```

Comment:
I was a little confused by the reassignment of `goal` to `fields` here (these will both refer to the same object)

Reply:
entirely so the linter wouldn't yell at me for mutating a parm with delete on line 114